app/assets/stylesheets/snowcrash/modules.scss

```diff
@@ -148,3 +148,10 @@ table.table-ajax-inputs {
     }
   }
 }
+
+#issues, #notes, #evidence, #tasks {
```

Comment:
@MattBudz 
- What is the purpose of this code?
- Why do we have to repeat it for each of these views?
- Looks like a behaviour we want in our layout (e.g. the content panel of every view)? In which case we could move it to the right file and reference the right class?

I'm concerned about us adding a new view further down the line and forgetting we have to hard-code it here too.

Reply:
This allows the collection to be vertically scrolled through in the sidebar. 
I agree that listing ID's is not the best approach. I have changed this to use a common class.